apps/extension/src/ui/domains/Account/AccountAdd/AccountAddQr/Scan.tsx

```diff
@@ -106,26 +107,29 @@ export const Scan = () => {
             enable={state.enable}
             error={!!state.cameraError}
             onScan={(scanned) => dispatch({ method: "onScan", scanned })}
-            onError={(error) =>
-              [
+            onError={(error) => {
+              const cameraErrors = [
                 "AbortError",
                 "NotAllowedError",
                 "NotFoundError",
                 "NotReadableError",
                 "OverconstrainedError",
                 "SecurityError",
-              ].includes(error.name)
-                ? dispatch({
-                    method: "setCameraError",
-                    error: error.name ?? error.message ?? "error",
-                  })
-                : dispatch({
-                    method: "setScanError",
-                    error: error.message.startsWith("Invalid prefix received")
-                      ? t("QR code is not valid")
-                      : error.message ?? "Unknown error",
-                  })
-            }
+              ]
+              if (cameraErrors.includes(error.name))
+                return dispatch({
+                  method: "setCameraError",
+                  error: error.name ?? error.message ?? "error",
+                })
+
+              dispatch({
+                method: "setScanError",
+                error: error.message.startsWith("Invalid prefix received")
+                  ? t("QR code is not valid")
+                  : error.message ?? "Unknown error",
+              })
+              console.error("QR code scanning error", error) // eslint-disable-line no-console
```

Comment:
Added this to make debugging similar problems easier in the future.

Reply:
might want to use `log.error` then :)

Reply:
nevermind, I thought log.error would ignore the DEBUG flag, but it doesn't